corehq/apps/api/resources/v0_5.py

```diff
@@ -802,12 +802,17 @@ def obj_get_list(self, bundle, **kwargs):
         return self.get_object_list(bundle.request)
 
     def get_object_list(self, request):
+        feature_flag = request.GET.get("feature_flag")
+        if feature_flag and feature_flag not in toggles.all_toggle_slugs():
+            raise BadRequest(f"{feature_flag!r} is not a valid feature flag")
         couch_user = CouchUser.from_django_user(request.user)
         results = []
         for domain in couch_user.get_domains():
             if not domain_has_privilege(domain, privileges.ZAPIER_INTEGRATION):
                 continue
             domain_object = Domain.get_by_name(domain)
+            if feature_flag and feature_flag not in toggles.toggles_dict(username=request.user, domain=domain):
```

Comment:
This is causing errors. `request.user` is a user object, not a username string, which this function is expecting. https://sentry.io/organizations/dimagi/issues/3294412535/?query=is%3Aunresolved+bad+type&statsPeriod=14d

I think you just need `request.user.username`

Reply:
Thanks @calellowitz. PR: https://github.com/dimagi/commcare-hq/pull/31662